class.jetpack-cli.php

```diff
@@ -1045,6 +1054,9 @@ public function partner_provision( $args, $named_args ) {
 		);
 
 		$url = sprintf( 'https://%s/rest/v1.3/jpphp/%d/partner-provision', $this->get_api_host(), $blog_id );
+		if ( ! empty( $named_args['host_tracking_id'] ) ) {
+			$url = esc_url_raw( add_query_arg( 'host_source_id', $named_args, $url ) );
```

Comment:
Seems like giving this a different name on WPCOM and Jetpack could be a source of bugs in the future. Any particular reason why it can't be called `host_tracking_id` in both places?

Also I think generally speaking `partner_tracking_id` or `partner_tracking_code` is better, because it lines up with the naming we give other partner things like "partner key" and "partner secret"

Reply:
No reason. I can make that change. 😄